changelogs/unreleased/gh-10853-async-commit-async-tx-after-sync-tx-synchro-timeout-crash.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/box
+
+* Fixed a crashed caused by asynchronously committing an asynchronous
+  transaction after a synchronous transaction with a timeout, the latter
+  timing out (gh-10853).
```

Comment:
Nit: It's probably the "Fixed ~~a crashed~~ the crash ...". And the meaning of the changelog eludes me, we probably should say it easier, smth like: "Fixed the crash caused by asynchronously committing an asynchronous transaction following a synchronous timed out transaction"

Reply:
Fixed.